draft-ietf-httpbis-client-hints.md

```diff
@@ -207,12 +207,15 @@ Therefore, features relying on this document to define Client Hint headers MUST
 
 Such features SHOULD take into account the following aspects of the information exposed: 
 
-  - Entropy
-    - Exposing highly granular data can be used to help identify users across multiple requests to different origins. Reducing the set of header field values that can be expressed, or restricting them to an enumerated range where the advertised value is close but is not an exact representation of the current value, can improve privacy and reduce risk of linkability by ensuring that the same value is sent by multiple users.
-  - Sensitivity
-    - The feature SHOULD NOT expose user sensitive information. To that end, information available to the application, but gated behind specific user actions (e.g. a permission prompt or user activation) SHOULD NOT be exposed as a Client Hint.
-  - Change over time
-    - The feature SHOULD NOT expose user information that changes over time, unless the state change itself is also exposed (e.g. through JavaScript callbacks).
+> {: vspace="0"}
+> Entropy:
```

Comment:
Do you really need the extra indentation?

Reply:
Maybe a better option is to get rid of the nested lists altogether and go with something like:
<img width="638" alt="Screenshot 2020-03-10 at 15 11 55" src="https://user-images.githubusercontent.com/786187/76320866-895da200-62e1-11ea-9afa-21646a6e7fa6.png">